### PR TITLE
Fix closed getter note (fixes #912)

### LINKED ISSUE #912: Note for ReadableStreamDefaultReader.close is confusing
The note for [ReadableStreamDefaultReader.close](https://streams.spec.whatwg.org/#default-reader-prototype) currently reads:

> The closed getter returns a promise that will be fulfilled when the stream becomes closed or the reader’s lock is released, or rejected if the stream ever errors.

@jakearchibald and I think that is not correct. 

@jakearchibald’s suggestion for the note:
> The closed getter returns a promise that will be fulfilled when the stream becomes closed. It rejects if the reader’s lock is released or the stream errors.

### PR DESCRIPTION
**Pull request by surma**
Took some inspiration from the writer’s `closed` getter :)

***
<a href="https://whatpr.org/streams/914.html" title="Last updated on Mar 29, 2018, 10:26 AM GMT (c2863de)">Preview</a> | <a href="https://whatpr.org/streams/914/cd63477...c2863de.html" title="Last updated on Mar 29, 2018, 10:26 AM GMT (c2863de)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1300,10 +1300,12 @@ lt="ReadableStreamDefaultReader(stream)">new ReadableStreamDefaultReader(<var>st
 <h5 id="default-reader-closed" attribute for="ReadableStreamDefaultReader" lt="closed">get closed</h5>
 
 <div class="note">
-  The <code>closed</code> getter returns a promise that will be fulfilled when the stream becomes closed or the
-  reader's lock is <a lt="release a read lock">released</a>, or rejected if the stream ever errors.
+  The <code>closed</code> getter returns a promise that will be fulfilled when the stream becomes closed, or rejected if
+  the stream ever errors or the reader's lock is <a lt="release a read lock">released</a> before the stream finishes
+  closing or the stream errors.
```
Comment (jakearchibald) [inline index.bs]: > or the stream errors.

I don't think you need to say this, as you've already made it clear that the promise rejects on error.

File: index.bs [hunk 2]
```diff
@@ -3531,7 +3533,7 @@ lt="WritableStreamDefaultWriter(stream)">new WritableStreamDefaultWriter(<var>st
 <div class="note">
   The <code>closed</code> getter returns a promise that will be fulfilled when the stream becomes closed, or rejected if
   the stream ever errors or the writer's lock is <a lt="release a write lock">released</a> before the stream finishes
-  closing.
+  closing or the stream errors.
```
Comment (jakearchibald) [inline index.bs]: As above.

File: index.bs (hunk 1 above)
Comment (surma) [reply index.bs]: lol

**Comment by ricea** [comment]
Travis CI doesn't seem to be happy. Since this isn't urgent, I (or someone else) can merge it once Travis CI is feeling better.

**Comment by surma** [comment]
I re-pushed my last commit to restart Travis. Seems like it worked :)

**Pull request merged**